PerDiem/PerDiem/Controllers/TransactionsViewController.m

```diff
@@ -26,6 +29,10 @@ - (void)viewDidLoad {
 
     self.edgesForExtendedLayout = UIRectEdgeNone;
     [self setupNavigationBar];
+
+    // @Todo Fetch Transactions
```

Comment:
Need to finish this, but wasn't sure how we were planning to organize the models for this common fetch.